src/goto-symex/build_goto_trace.h

```diff
@@ -17,14 +17,26 @@ Date: July 2005
 #include "symex_target_equation.h"
 #include "goto_symex_state.h"
 
-// builds a trace that stops at first failing assertion
+/// Build a trace by going through the steps of \p target and stopping at the
+/// first failing assertion
+/// \param target: SSA form of the program
+/// \param prop_conv: solver from which to get valuations
+/// \param ns: namespace
+/// \param[out] goto_trace: trace to which the steps of the trace get appended
```

Comment:
Nit pick: the coding standard suggests there should be a space between `param` and `[out]`